src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -5566,6 +5568,247 @@ void GetVRFInputData(EnergyPlusData &state, bool &ErrorsFound)
     }
 }
 
+void CheckVRFTUNodeConnections(EnergyPlusData &state, int const VRFTUNum, bool &ErrorsFound)
+{
+
+    std::string const cTerminalUnitType("ZoneHVAC:TerminalUnit:VariableRefrigerantFlow");
```

Comment:
@lefticus could you document in a GitHub comment here the best practice for local variable strings?  We have discussed them on calls and maybe chats, but it would be good to lock it down here.  IIRC it's going to be a constexpr string_view, but I can't recall all the implications.

Reply:
@lefticus, use constexpr static std::string_view cTerminalUnitType("string"); here?

Reply:
@Myoldmopar @rraustad I wrote up a full explanation of what is best and what the implications may or may not be on this other PR: 

https://github.com/NREL/EnergyPlus/pull/8663